packages/api/src/webhooks/cannon.ts

```diff
@@ -615,7 +617,10 @@ async function storeResponse(
         method: params.method,
         headers: params.headers,
       },
-    });
+      sharedSecret: sharedSecret,
+    };
+    await db.webhookResponse.create(webhookResponse);
+    return webhookResponse;
   } catch (e) {
```

Comment:
Perhaps we should rethrow this error now, otherwise the API will respond with a `200` and an "undefined" body

Reply:
I moved the catch into the main firehook function where it's still needed. hopefully that looks good to you https://github.com/livepeer/studio/pull/2021/commits/9490476346cd7d70ae71e0167b66582e88c02616